scripts/sanity_check.sh

```diff
@@ -386,15 +386,23 @@ check_hosts() {
         local DEVICE_FLOWINCOUNT=$(redis-cli zcount flow:conn:in:$DEVICE_MAC -inf +inf)
         local DEVICE_FLOWOUTCOUNT=$(redis-cli zcount flow:conn:out:$DEVICE_MAC -inf +inf)
 
-        local COLOR=""
+        local COLOR="\e[39m"
         local UNCOLOR="\e[0m"
         if [[ $DEVICE_ONLINE == "yes" && $DEVICE_MONITORING == 'true' && $DEVICE_B7_MONITORING == "false" ]] &&
           ! is_firewalla $DEVICE_IP && ! is_router $DEVICE_IP && is_simple_mode; then
             COLOR="\e[91m"
         elif [ $DEVICE_FLOWINCOUNT -gt 2000 ] || [ $DEVICE_FLOWOUTCOUNT -gt 2000 ]; then
             COLOR="\e[33m" #yellow
-        elif [ $DEVICE_ONLINE = "no" ]; then
-            COLOR="\e[2m" #dim
+        fi
+
+        local DEVICE_MAC_COLOR="$COLOR"
+        if [[ $DEVICE_MAC =~ ^.[26AEae].*$ ]]; then
+          DEVICE_MAC_COLOR="\e[95m"
+        fi
+
+        if [ $DEVICE_ONLINE = "no" ]; then
```

Comment:
There is a logic change here, where color yellow could be overridden by color dim after the change.